bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/container.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FullscreenButtonComponent from './component';
+import { toggleFullScreen } from '/imports/ui/components/nav-bar/settings-dropdown/service';
+
+const FullscreenButtonContainer = props => <FullscreenButtonComponent {...props} />;
+
+export default (props) => {
+  const handleToggleFullScreen = ref => toggleFullScreen(ref);
+  return (
+    <FullscreenButtonContainer {...props} {...{ handleToggleFullScreen }} />
```

Comment:
```suggestion
    <FullscreenButtonContainer  { ...props, ...{ handleToggleFullScreen }} />
```

Reply:
This seems to cause a syntax error.
![image](https://user-images.githubusercontent.com/22058534/57856039-259a4380-77ba-11e9-86d9-055ea70a112b.png)